src/contract/deploy.rs

```diff
@@ -264,4 +258,63 @@ mod tests {
         assert_eq!(error.to_string(), DeployError::EmptyBytecode.to_string());
         transport.assert_no_more_requests();
     }
+
+    #[test]
+    fn deploy() {
+        let mut transport = TestTransport::new();
+        let web3 = Web3::new(transport.clone());
+
+        let mut artifact = Artifact::empty();
+        let bytecode = "0x00";
+        artifact.bytecode = Bytecode::from_hex_str(bytecode).unwrap();
+        let linker = Linker::new(artifact);
+
+        // eth_accounts
+        let account = addr!("0x9876543210987654321098765432109876543210");
+        transport.add_response(json!([account]));
+
+        // eth_sendTransaction
+        let transaction_hash =
+            hash!("0x4242424242424242424242424242424242424242424242424242424242424242");
+        transport.add_response(json!(transaction_hash));
+
+        // eth_blockNumber
+        transport.add_response(json!("0x0"));
+
+        // eth_getTransactionReceipt
+        let contract_address = addr!("0x9876543210987654321098765432109876543210");
+        let block_hash =
+            hash!("0x4242424242424242424242424242424242424242424242424242424242424242");
+        let mut logs_bloom = "0x".to_string();
```

Comment:
```suggestion
        let logs_bloom = H2048::repeat_byte(0);
```
https://docs.rs/web3/0.10.0/web3/types/struct.H2048.html#method.repeat_byte

Reply:
This can probably be fixed in other locations in the code.